0003-siren-prototype-accept-decline.adoc

```diff
@@ -0,0 +1,88 @@
+# Siren Prototype for comit-rs HTTP API - Accept/Decline acions
+
+* Authors: Franck Royer
+* Date: 2019-05-02
+
+Tracking issue: [comit-rs#917](https://github.com/comit-network/comit-rs/issues/917)
+
+## Context
+
+In order to move forward easily with Bobtimus prime we should define the siren json document of how a IN_PROGRESS response look like in each state of accept and decline, i.e. when you call GET /swaps/ for Bob.
+
+Siren repository: https://github.com/kevinswiber/siren/
+
+The prototype must conform to [Siren JSON schema](https://github.com/kevinswiber/siren/blob/master/siren.schema.json)
+
+## Research
+
+The prototype:
+
+```json
+{
+  "class": ["swaps"],
+  "entities": [
+    {
+      "class": ["swap"],
+      "rel": [],
+      "properties": {
+        "role": "Alice",
+        "protocol": "rfc003",
+        "status": "IN_PROGRESS",
+        "parameters": {
+          "alpha_asset": {
+            "name": "bitcoin",
+            "quantity": "100000000"
+          },
+          "alpha_ledger": {
+            "name": "bitcoin",
+            "network": "regtest"
+          },
+          "beta_asset": {
+            "name": "ether",
+            "quantity": "10000000000000000000"
+          },
+          "beta_ledger": {
+            "name": "ethereum",
+            "network": "regtest"
+          }
+        }
+      },
+      "links": [
+        {"rel": ["self"], "href": "/swaps/rfc003/399e8ff5-9729-479e-aad8-49b03f8fc5d5"},
+        {
+          "rel": ["human-protocol-spec"],
+          "href": "https://github.com/comit-network/RFCs/blob/master/RFC-003-SWAP-Basic.md"
+        }
+      ],
+      "actions": [
+        {
+          "name": "accept",
+          "title": "Accept",
+          "method": "POST",
+          "href": "/swaps/rfc003/399e8ff5-9729-479e-aad8-49b03f8fc5d5/accept",
+          "type": "application/json",
+          "class": ["comit-action"],
+          "fields": [
+            {
+              "name": "beta_ledger_refund_identity", "title": "Refund address on Ethereum", "type": "text",
```

Comment:
Is type actually text here?

Reply:
yes, siren defines a list of acceptable values.